rq/registry.py

```diff
@@ -352,3 +411,9 @@ def clean_registries(queue):
                                  job_class=queue.job_class,
                                  serializer=queue.serializer)
     registry.cleanup()
+
+    registry = QueuedJobRegistry(name=queue.name,
+                                 connection=queue.connection,
+                                 job_class=queue.job_class,
+                                 serializer=queue.serializer)
+    registry.requeue_stuck_jobs()
```

Comment:
I'm not sure if we should automatically requeue stuck jobs because:
1. This operation could be heavy depending on the number of jobs you have in the registry
2. If requeued, jobs that are time sensitive could lead to unwanted outcome